fe/fe-core/src/test/java/org/apache/doris/nereids/stats/FilterEstimationTest.java

```diff
@@ -71,7 +71,9 @@ public void test1() {
         FilterEstimation filterEstimation = new FilterEstimation(stat);
         StatsDeriveResult expected = filterEstimation.estimate(or);
         Assertions.assertTrue(
-                Precision.equals((0.5 * 0.1 + 0.1 - 0.5 * 0.1 * 0.1) * 1000, expected.getRowCount(), 0.01));
+                Precision.equals((0.5 * 0.1 + FilterEstimation.DEFAULT_EQUALITY_COMPARISON_SELECTIVITY
```

Comment:
plz add comment to explain these magic numbers

Reply:
done